vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -101,6 +101,14 @@ jobs:
           testRunTitle: "Windows (Attempt $(System.JobAttempt))"
         continueOnError: true
         condition: always()
+        
+      - task: ComponentGovernanceComponentDetection@0
+        displayName: Component Governance Detection
+        inputs:
+          scanType: 'Register'
+          verbosity: 'Verbose'
+          alertWarningLevel: 'Low' # The task will present a warning, but will not cause the build to fail
+        condition: always()
```

Comment:
This build task was automatically inserted for us, but that version only ran this check if the build succeeded. For flakey pipelines, that means we would be subject to out-of-date component governance alerts until we stabilize the pipeline